### ISSUE TITLE
New release required

### ISSUE THREAD

**Issue by Rxbsxn** [simplabs/ember-simple-auth#2247]
Hello there,
I see #2237 was merged to the master branch, is there any chance to have it released soon? It causes a lot of deprecation warnings in my application, thanks!

**Comment by rjschie** [comment]
It would also be nice to get the bug fix for the config `routeAfterAuthentication` released (https://github.com/simplabs/ember-simple-auth/pull/2215)

**Comment by josemarluedke** [comment]
I'm also looking forward to a new release containing #2237.

**Comment by PoslinskiNet** [comment]
@Turbo87 @marcoow any thoughts on this one? Would be nice not to rely on forks 🙏 .

**Comment by ashleysommer** [comment]
I agree, this really needs a new release.
I today I followed the Walkthrough in the README file, only to find out after I finished that large portions of the README were modified to adapt to the changes in #2198, but there has not yet been a release with those changes incorporated.

Edit: Just saw that [3.1.0-beta.1](https://github.com/simplabs/ember-simple-auth/releases/tag/3.1.0-beta.1) was released today. Thanks. I'll update to the beta.
